diff --git a/module.d.ts b/module.d.ts
index 3d04c3303f0eae7b23af318b68bc101ad6fe2613..d7797b36b268f7d4eca81dda508cc17b9f10d159 100644
--- a/module.d.ts
+++ b/module.d.ts
@@ -211,7 +211,8 @@ declare module "module" {
             /**
              *  An object whose key-value pairs represent the assertions for the module to import
              */
-            importAssertions: ImportAssertions;
+            importAssertions?: ImportAssertions;
+            importAttributes?: ImportAssertions;
         }
         interface LoadFnOutput {
             format: ModuleFormat;
