diff --git a/lib/windowsPtyAgent.js b/lib/windowsPtyAgent.js
index 954acef4998a78a907c7059a0b22b48e6836ca21..76e27677d2c1842fbb7e8c4304aff9a1e52ad78a 100644
--- a/lib/windowsPtyAgent.js
+++ b/lib/windowsPtyAgent.js
@@ -150,7 +150,7 @@ var WindowsPtyAgent = /** @class */ (function () {
         // Tell the agent to kill the pty, this releases handles to the process
         if (this._useConpty) {
             this._getConsoleProcessList().then(function (consoleProcessList) {
-                consoleProcessList.forEach(function (pid) {
+                (consoleProcessList || []).forEach(function (pid) {
                     try {
                         process.kill(pid);
                     }